src/pgrn-query-expand.c

```diff
@@ -502,7 +503,19 @@ pgroonga_query_expand(PG_FUNCTION_ARGS)
 	}
 	tableOID = DatumGetObjectId(tableOIDDatum);
 	currentData.table = RelationIdGetRelation(tableOID);
-
+#ifdef PGRN_HAVE_RELKIND_HAS_TABLE_AM
+	if(!RELKIND_HAS_TABLE_AM(currentData.table->rd_rel->relkind))
+#else
+	if((currentData.table->rd_rel->relkind != RELKIND_RELATION)
+	   && (currentData.table->rd_rel->relkind != RELKIND_TOASTVALUE)
+	   && (currentData.table->rd_rel->relkind != RELKIND_MATVIEW))
+#endif
+	{
+		PGrnCheckRC(GRN_INVALID_ARGUMENT,
+					"%s the value of table_name argument isn't table object: <%s>",
```

Comment:
```suggestion
					"%s the specified table isn't table: <%s>",
```

Could you also show what is the specified object for easy to debug? (Can we stringify `relkind`?)